src/component/admin/groups/GroupForm/GroupFormUsersTable/GroupFormUsersTable.tsx

```diff
@@ -11,12 +11,28 @@ import { VirtualizedTable } from 'component/common/Table';
 import { useFlexLayout, useSortBy, useTable } from 'react-table';
 import { sortTypes } from 'utils/sortTypes';
 import { UserAvatar } from 'component/common/UserAvatar/UserAvatar';
+import theme from 'themes/theme';
 
 interface IGroupFormUsersTableProps {
     users: IGroupUser[];
     setUsers: React.Dispatch<React.SetStateAction<IGroupUser[]>>;
 }
 
+const useHiddenColumns = (): string[] => {
+    const isMediumScreen = useMediaQuery(theme.breakpoints.down('md'));
+
+    return useMemo(() => {
```

Comment:
What is the value of useMemo here? In my opinion useMemo should only be used for large expensive computation time calculations. If we use it for anything else it's just overhead that doesn't give any tangible benefit and increases the memory footprint.

Reply:
Found a way around useMemo